Tabloid/Repositories/UserProfileRepository.cs

```diff
@@ -1,6 +1,10 @@
-﻿using Microsoft.Extensions.Configuration;
+﻿using Microsoft.Data.SqlClient;
+using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
 using Tabloid.Models;
 using Tabloid.Utils;
+using System.Linq;
+using System.Runtime.InteropServices;
```

Comment:
This line:
using System.Runtime.InteropServices;
isn't necessary (probably your tab-completion being a little overzealous)

Reply:
I removed it and pushed it back up.